packages/react-instantsearch/src/connectors/connectHierarchicalMenu.js

```diff
@@ -133,7 +133,12 @@ export default createConnector({
   },
 
   getProps(props, state, search) {
-    const {id, showMore, limitMin, limitMax} = props;
+    const {attributes, showMore, limitMin, limitMax} = props;
+
+    if (typeof attributes === 'undefined' || !attributes.length)
+      throw new Error('attributes props should at least have one element');
```

Comment:
It doesn't check the number of items, but it still better! I will move to checking with proptypes.